tests/tools/assigner/actions/balancemodules/test_even.py

```diff
@@ -143,3 +143,9 @@ def test_process_cluster_4broker(self):
         action = ActionBalanceEven(self.args, self.cluster)
         action.process_cluster()
         assert self.is_cluster_even(skip_topics=['testTopic3'])
+
+    def test_process_cluster_big(self):
+        self.cluster = set_up_cluster_big()
+
+        action = ActionBalanceEven(self.args, self.cluster)
+        action.process_cluster()
```

Comment:
This test doesn't actually validate that the cluster layout that is produced is even. It only checks that the code ran without error.

Reply:
Ah yeah, sorry, forgot to add a proper test case for it, we'll update here